libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -502,71 +718,76 @@ struct LocalVideoTrackInitConfig {
+/// Configuration for creating a new audio transceiver interop wrapper.
+struct mrsAudioTransceiverConfig {
+  /// Transceiver name, for pairing with remote peer's one. This is normally
+  /// extracted from the receiver's stream ID #0 if the transceiver was created
+  /// for a remote track, or is determined by the user-provided value if created
+  /// with |AddAudioTransceiver()|.
+  const char* name{};
+
+  /// Zero-based media line index for the transceiver. This is the index of the
+  /// m= line in the SDP offer/answer as determined when adding the transceiver.
+  /// This is provided by the implementation and is immutable (since we don't
+  /// support stopping transceivers, so m= lines are not recycled).
+  int mline_index = -1;
+
+  /// Initial desired direction when the transceiver is created. If the
+  /// transceiver is created by the implementation, this is generally set to the
+  /// current value on the implementation object, to keep the interop wrapper in
+  /// sync.
+  mrsTransceiverDirection initial_desired_direction =
+      mrsTransceiverDirection::kSendRecv;
 };
 
-/// Add a local video track from a custom video source external to the
-/// implementation. This allows feeding into WebRTC frames from any source,
-/// including generated or synthetic frames, for example for testing.
-/// The track source initially starts as capuring. Capture can be stopped with
-/// |mrsExternalVideoTrackSourceShutdown|.
-/// This returns a handle to a newly allocated object, which must be released
-/// once not used anymore with |mrsLocalVideoTrackRemoveRef()|.
-MRS_API mrsResult MRS_CALL
-mrsPeerConnectionAddLocalVideoTrackFromExternalSource(
-    PeerConnectionHandle peerHandle,
-    const char* track_name,
-    ExternalVideoTrackSourceHandle source_handle,
-    const LocalVideoTrackFromExternalSourceInitConfig* config,
-    LocalVideoTrackHandle* track_handle) noexcept;
-
-/// Remove a local video track from the given peer connection and destroy it.
-/// After this call returned, the video track handle is invalid.
-MRS_API mrsResult MRS_CALL mrsPeerConnectionRemoveLocalVideoTrack(
-    PeerConnectionHandle peer_handle,
-    LocalVideoTrackHandle track_handle) noexcept;
-
-/// Remove all local video tracks backed by the given video track source from
-/// the given peer connection and destroy the video track source.
-/// After this call returned, the video track source handle is invalid.
-MRS_API mrsResult MRS_CALL mrsPeerConnectionRemoveLocalVideoTracksFromSource(
-    PeerConnectionHandle peer_handle,
-    ExternalVideoTrackSourceHandle source_handle) noexcept;
-
-/// Add a local audio track from a local audio capture device (microphone) to
-/// the collection of tracks to send to the remote peer.
-MRS_API mrsResult MRS_CALL
-mrsPeerConnectionAddLocalAudioTrack(PeerConnectionHandle peerHandle) noexcept;
+/// Configuration for creating a new video transceiver interop wrapper.
+struct mrsVideoTransceiverConfig {
```

Comment:
nit: merge with above, just use mrsTransceiverConfig?

Reply:
Yes I was hesitating if we want to keep it for later extension, but since the implementation mostly uses a single transceiver class I guess it's unlikely they diverge.

Reply:
Fixed as `mrsTransceiverInitConfig` to emphasize on creation of a new transceiver.

Reply:
Actually, changed to `mrsTransceiverWrapperInitConfig` because this one is for creating a wrapper. The one for actually creating a transceiver is now merged into `mrsTransceiverInitConfig`.